lib/govuk_content_schemas/allowed_document_types.yml

```diff
@@ -149,9 +138,7 @@
 - travel_advice_index
 - unpublishing
 - utaac_decision
-- vanish
```

Comment:
As far as I know, `vanish` is still being used in the Publishing API. I haven't checked any of the other document types yet.

Reply:
Thanks, updated.